custom/icds_reports/tasks.py

```diff
@@ -985,5 +986,8 @@ def create_mbt_for_month(state_id, month):
 
 @task(queue='background_queue')
 def _bust_awc_cache():
+    create_datadog_event('redis: delete dashboard keys', 'start')
     reach_keys = cache.keys('*cas_reach_data*')
```

Comment:
Do you think using keys could be causing issues?

https://redis.io/commands/keys

> Warning: consider KEYS as a command that should only be used in production environments with extreme care. It may ruin performance when it is executed against large databases. This command is intended for debugging and special operations, such as changing your keyspace layout